src/core/cmd.rs

```diff
@@ -14,6 +14,10 @@ pub enum Command {
     Quit,
     /// Save the current file buffer.
     Save(Option<ViewId>),
+    /// Backspace
+    Back(),
```

Comment:
Is there any reason these can't be unit struct variants? e.g. Back & Delete(no parenthesis)

Reply:
Nope. Don't know what I was thinking.